### PR TITLE
Update apollo-link-rest.md

### PR DESCRIPTION
**Pull request by tobiasl-linktree**
`body` isn't defined in the scope of the function of the example, and `data` isn't used, I assume that's because data should be named body?

### Checklist:

- [ ] If this PR is a new feature, please reference an issue where a consensus about the design was reached (not necessary for small changes)
- [ ] Make sure all of the significant new logic is covered by tests

### CODE REVIEW

**Review by jpvajda — COMMENTED** [review]
Thanks for pointing this, out I think the `data` parameter is referenced from some of the code samples above in this doc, and I'm curious if it should be

```
 defaultSerializer: (data: any, headers: Headers) => {
    const formData = new FormData();
    for (let key in data) {
      formData.append(key, data[key]);
    }
    headers.set("Content-Type", "x-www-form-encoded")
    return {body: formData, headers};
  }

it looks like @hwillson [committed this ](https://github.com/apollographql/apollo-client/commit/e8d78769130be69c33742edd3595f04a3f5bc324)

So let's see if he can provide some guidance here.

**Comment by hwillson** [comment]
@jpvajda - the suggested solutions from @tobiasl-linktree or yourself both work, so I'm 👍 with whichever one you're interested in merging (but yes this is a documentation bug). Thanks!

File: docs/source/api/link/apollo-link-rest.md [hunk 1]
```diff
@@ -383,7 +383,7 @@ const link = new RestLink({
       bodySnippet...
     }
   },
-  defaultSerializer: (data: any, headers: Headers) => {
+  defaultSerializer: (body: any, headers: Headers) => {
```
Comment (jpvajda) [inline docs/source/api/link/apollo-link-rest.md:386]:
```suggestion
  defaultSerializer: (data: any, headers: Headers) => {
```

File: docs/source/api/link/apollo-link-rest.md [hunk 2]
```diff
@@ -383,7 +383,7 @@ const link = new RestLink({
...
     const formData = new FormData();
     for (let key in body) {
```
Comment (jpvajda) [inline docs/source/api/link/apollo-link-rest.md:388]:
```suggestion
    for (let key in data) {
```

Comment (jpvajda) [inline docs/source/api/link/apollo-link-rest.md:388]:
```suggestion
    for (let key in data) {
```

**Comment by jpvajda** [comment]
This got a little messy with all the suggested commits, so I'm closing it out and made a fresh PR 🥦 

https://github.com/apollographql/apollo-client/pull/9711

**Pull request closed without merging**